osu.Game.Rulesets.Taiko.Tests/osu.Game.Rulesets.Taiko.Tests.csproj

```diff
@@ -11,5 +11,6 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\osu.Game.Rulesets.Taiko\osu.Game.Rulesets.Taiko.csproj" />
+    <ProjectReference Include="..\osu.Game.Tests\osu.Game.Tests.csproj" />
```

Comment:
This is intentional - the added reference allows using `BeatmapImportHelper.LoadOszIntoOsu()` in the test, which resides in `osu.Game.Tests`.

Yes I considered moving that class to `osu.Game`, but then I also have to move its dependency `TestResources`, and then I probably need to move all of the resources _that_ references to the main game project, so not doing that.

Yes I considered not relying on an import in the test, and tried making it work with an empty beatmap, and it doesn't work for unknown reasons that I don't want to waste an hour debugging. A non-empty beatmap that hasn't been imported via standard import flows dies on a realm assertion.

So if this is deemed unacceptable then I'll likely just force-push https://github.com/ppy/osu/pull/28995/commits/e2fe1935a92943c5e505bf82cf76e4e59ab93298 entirely out of this diff.

Reply:
I'm okay with this FWIW